### ISSUE TITLE
[Accessibility] The tick mark besides the papers are not labelled; Screen reader ignores them completely.

### ISSUE THREAD

**Issue by SushantKafle** [acl-org/acl-2020-virtual-conference#344]
The tick mark besides the papers are not labelled, so SR ignores them completely.

Solution: Provide "aria-label" for the div. Make sure to indicate whether the div is selected or un-selected -- which could be done through the label.

![Screenshot 2020-06-30 at 9 25 59 AM](https://user-images.githubusercontent.com/4416671/86323871-7bb28c00-bbf2-11ea-893c-d35493b5d58d.png)
